src/state/stake/stake-constants.ts

```diff
@@ -394,6 +398,64 @@ const AURORA_POOLS: StakingTri[] = [
     noTriRewards: true,
     inStaging: false,
     doubleRewardToken: BBT[ChainId.AURORA]
+  }),
+  // Needed to add the this pool due to some functions and features breaking when jumping from ID 24 to 26. 
+  // TODO:  Will be replaced by stable farm pool in stable farms PR.
```

Comment:
So that we remember this, please create a card for this in Notion and add it in `next up`

Reply:
https://www.notion.so/54056ca820b44017b2ec73620324ebff?v=1e2ac750ac35462a8f524b62f8234ab2&p=5f51e34078544403949730dafdb9babd


🙏